app/lib/errors.js

```diff
@@ -68,6 +68,12 @@ class PostcodeNotFoundError extends PostcodesioHttpError {
     super(404, "Postcode not found");
   }
 }
+
+class PostcodeNotScottishError extends PostcodesioHttpError {
```

Comment:
Can we change to PostcodeNotSpdError or something to that effect

Reply:
Done! :)